ViteMaDose/ViewModels/CentresList/CentresListFilterOption.swift

```diff
@@ -0,0 +1,49 @@
+// Software Name: vitemadose-ios
+// SPDX-FileCopyrightText: Copyright (c) 2021 CovidTracker.fr
+// SPDX-License-Identifier: GNU General Public License v3.0 or later
+//
+// This software is distributed under the GPL-3.0-or-later license.
+//
+
+import Foundation
+
+/// Available filtering options for centres list
+public enum CentresListFilterOption: Int, CaseIterable {
+
+    /// FIlter nothing, keep all centers
+    case allDoses
+    /// Keep only centres with kids first doses
+    case kidsFirstDoses
+
+    /// Vaccine type "Moderna"
+    case vaccineTypeModerna
+    /// Vaccine type "Pfizer"
+    case vaccineTypePfizer
+    /// Vaccine type "ARN-m"
+    case vaccineTypeARNm
+    /// Vaccine type "Janssen"
+    case vaccineTypeJanssen
+    /// Vaccine type "Novavax"
+    case vaccineTypeNovavax
+
+    public init?(rawValue: Int) {
+        switch rawValue {
```

Comment:
Non, l'idée c'est de faire une boucle sur `allCases` pour éviter toutes ces lignes :

for value in allCases {
  if rawValue == value.rawValue {
    return value
  }
}
return nil

(A vérifier sur Xcode, je l'ai écrit à la main via mon téléphone)